flink-libraries/flink-table/pom.xml

```diff
@@ -146,6 +147,12 @@ under the License.
 			<artifactId>flink-test-utils_${scala.binary.version}</artifactId>
 			<version>${project.version}</version>
 			<scope>test</scope>
+			<exclusions>
```

Comment:
Why does `flink-test-utils` even have a Guava dependency? Can we fix that?

Reply:
flink-test-utils depends on curator-test which depends on guava.